app/views/github_oauth.py

```diff
@@ -36,6 +39,7 @@ def github_authenticate(request):
         )
 
     if user is None:
+        logging.error(f"{github_user_name}'s account not created.")
```

Comment:
Can also be rephrased as - 
Github account for {github_user_name} was not created.